big_tests/tests/mod_event_pusher_rabbit_SUITE.erl

```diff
@@ -40,13 +41,15 @@
          {amqp_port, 5672},
          {amqp_username, <<"guest">>},
          {amqp_password, <<"guest">>},
-         {presence_exchange, ?PRESENCE_EXCHANGE},
-         {chat_msg_exchange, ?CHAT_MSG_EXCHANGE},
-         {chat_msg_sent_topic, ?CHAT_MSG_SENT_TOPIC},
-         {chat_msg_recv_topic, ?CHAT_MSG_RECV_TOPIC},
-         {groupchat_msg_exchange, ?GROUP_CHAT_MSG_EXCHANGE},
-         {groupchat_msg_sent_topic, ?GROUP_CHAT_MSG_SENT_TOPIC},
-         {groupchat_msg_recv_topic, ?GROUP_CHAT_MSG_RECV_TOPIC}
+         %% enables publisher one-to-one confirms; disabled by default
+         %% {confirms_enabled, true},
+         {presence_exchange, [{name, ?PRESENCE_EXCHANGE}]},
+         {chat_msg_exchange, [{name, ?CHAT_MSG_EXCHANGE},
+                              {sent_topic, ?CHAT_MSG_SENT_TOPIC},
+                              {recv_topic, ?CHAT_MSG_RECV_TOPIC}]},
+         {groupchat_msg_exchange, [{name, ?GROUP_CHAT_MSG_EXCHANGE},
```

Comment:
I was thinking about that but I decided to stick to `topic` word because routing keys we use take advantage of topic syntax `user_N@host.chat_msg_sent` and you can only set the latest part (after user jid). Does it make sense?